backend/server.ts

```diff
@@ -19,53 +21,45 @@ app.use(cors());
 // Serve static files from the "Frontend" directory
 app.use(express.static('Frontend'));
 
-// Set up AWS S3
-const s3 = new AWS.S3({
-    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
-    region: 'us-east-2',
-});
-
 const storage = multer.memoryStorage(); // Store the file in memory
 const upload = multer({ storage: storage });
 
-app.get('/upload-page', (req, res) => {
+app.get('/upload-page', (req, res) => { // This is just for testing purposes
     res.sendFile(path.join(__dirname, '../Frontend/testwebsite.html'));
 });
 
-app.post('/upload', upload.single('packageContent'), async (req, res, next) => {
+app.post('/package', upload.single('packageContent'), async (req, res) => {
     try {
         if (!req.file) {
```

Comment:
Idk if I'm misunderstanding this but remember that "On package upload, either Content or URL should be set. If both are set, returns 400." So you shouldn't return an error immediately if the Content doesn't exist because they didn't upload a file. Just check if both of them are set or if none of them are set for the error checking.